content/en/docs/troubleshooting.md

```diff
@@ -139,6 +139,47 @@ SHA1 Thumbprint: xxx
 
 Check the `Subject` info in the output. If it is the identity that signs the artifact, you need to add the `Subject` info into `trustedIdentities`.
 
+## When I verify an artifact, I get the error 'signature verification failed for all the signatures associated with IMAGE'
+
+This may result from one of the following issues:
+
+- Not having a trust store configured or the trust store is not configured correctly.
+- Trust store is not readable or the certificates stored in the trust store are not readable
+- no certificates were stored in trust store
+
+Rerun the verify command with `--verbose` to get more details. 
+
+The following example shows the trust store `mystore` has not be configured correctly and can't be found:
+
+```console
+$ notation verify $IMAGE -v
+...
+ERRO authenticity validation failed. Failure reason: error while loading the trust store, "/home/USER/.config/notation/truststore/x509/ca/mystore" does not exist
+WARN Signature sha256:11223344556677889900aabbccddeeff11223344556677889900aabbccddeeff failed verification with error: error while loading the trust store, "/home/USER/.config/notation/truststore/x509/ca/mystore" does not exist
+Error: signature verification failed for all the signatures associated with localhost:5001/net-monitor@sha256:000999888777666555444333222111fffeeedddcccbbbbaaa000999888777666
+```
+
+The following example shows the certificate `mystore.crt` in the `mystore` trust store could not be accessed:
```

Comment:
```suggestion
The following example shows the usecase where notation couldn't read the certificate `mystore.crt` stored in `mystore` trust store:
```

Reply:
i think the current wording is fine as is